doc/cli.markdown

```diff
@@ -1857,7 +1865,7 @@ Examples:
 	$ balena tunnel myApp -p 22222:0.0.0.0:222
 	
 	# multiple port tunnels can be specified at any one time
-	$ balena tunnel myApp -p 8080:3000 -p 8081:9000
+	$ balena tunnel myApp -p 8080:3000 8081:9000
```

Comment:
Are both these formats supported? `-p 8080:3000 -p 8081:9000` / `-p 8080:3000 8081:9000` 

I wonder if the latter might be confusing for people who have seen the comma seperated convention for we have adopted for multi arguments?